boards/DIM/Test/Src/Test_StateMachine.cpp

```diff
@@ -506,17 +474,24 @@ TEST_F(DimStateMachineTest, dcm_board_status_led_control_with_multiple_errors)
     // If the error table contains critical and non-critical errors
     // simultaneously, the critical error should take precedence and turn the
     // DCM LED red rather than blue
+    // TODO: JSON2CAN
     //    App_SharedErrorTable_SetError(error_table, DCM_AIR_SHUTDOWN_MISSING_HEARTBEAT, true);
     //    App_SharedErrorTable_SetError(error_table, DCM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ, true);
+    App_CanRx_DCM_Warnings_MissingHeartbeat_Update(true);
```

Comment:
Is MissingHeartbeat a Warning or a Fault? It was previously an AIR_SHUTDOWN message, should it be promotoed to a  a fault and should we be disconnecting AIRS when it occurs?